src/InOneWeekend/camera.h

```diff
@@ -113,8 +113,8 @@ class camera {
         }
 
         vec3 unit_direction = unit_vector(r.direction());
-        auto a = 0.5*(unit_direction.y() + 1.0);
-        return (1.0-a)*color(1.0, 1.0, 1.0) + a*color(0.5, 0.7, 1.0);
+        auto t = 0.5*(unit_direction.y() + 1.0);
+        return (1.0-t)*color(1.0, 1.0, 1.0) + t*color(0.5, 0.7, 1.0);
```

Comment:
Change of variable name requires matching changes (eleven instances) in book 1.

Reply:
AHHHHHHHHH

Another casualty of the merge. I had this correct before the merge

Reply:
Op. Nope. This was a bug.
Fixing.

Reply:
OH GOD. I remember this F bug.

We're using `a` to represent the linear interpolation between white at y=0 and blue at y=1.

I would strongly prefer that we use `t` for this lerp. `t` is much more common in the literature. But we can't because we use `t` because that's reserved for the ray (`point = origin + t*dir`)

I'd also prefer to use `x` as it's much more natural to think as a lerp along two points on a grid, but as we're interpolating between two points in 3d space, `x` would be super confusing (as the line could be in any direction)

Ugh. We could probably go with like, `w` or something. But for now I'm running back into my cave and reverting everything to `auto a`